ecosystem/typescript/sdk/src/plugins/fungible_asset_client.ts

```diff
@@ -0,0 +1,113 @@
+import { AptosAccount } from "../account";
+import { RawTransaction } from "../aptos_types";
+import * as Gen from "../generated/index";
+import { OptionalTransactionArgs, Provider } from "../providers";
+import { TransactionBuilderRemoteABI } from "../transaction_builder";
+import { MaybeHexString, HexString } from "../utils";
+
+export class FungibleAssetClient {
```

Comment:
Did you consider combining this with the existing coin client at all?

Reply:
you mean https://github.com/aptos-labs/aptos-core/pull/8294/files#diff-dca373a5371526fcac7c2fb7088983125df962743aea81253771be2ea19dd87bR36 ?

Reply:
Oh weird sorry. I clicked on a link from email so I was accidentally only looking at changes from the latest commit